numerical/interpolation.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+from numerical import splines
+from numerical.utils.interpolation import repeat_args
+
+
+def interpolate(values, grid, batch_size=16):
+    """ Builds function which is an interpolation function on nodes with computer values in these nodes.
+
+    Args:
+        values: list of function values in grid nodes.
+        grid: points where function was calculated used np.meshgrid function with parameter 'indexing='ij''.
+        batch_size: int, batch size for interpolation process.
+
+    Returns:
+        interpolated function.
+    """
+    # linear interpolation will be used as a basis function
+    bfunc = splines.linear
```

Comment:
This one could be a funct arg. Lets do not limit a user o use only this kind of splines for intrpolatiin

Reply:
for now, it works just for linear. `bfunc` will be a parameter in the future, but not now

Reply:
Raise an issue then